src/@apollo/client/core/ApolloClient__Core_ApolloClient.res

```diff
@@ -313,12 +313,13 @@ module Js_ = {
   ) => Js.Promise.t<FetchResult.Js_.t<'jsData>> = "mutate"
 
   // onClearStore(cb: () => Promise<any>): () => void;
+  // TODO: named arguments to binding? - AxM
```

Comment:
Did you want to discuss this?

Reply:
Was just curious myself, doesnt need your time. ill remove it.